### ISSUE TITLE
Bunch of missing generated header files in Visual Studio 2015

### ISSUE THREAD

**Issue by kungfooman** [google/flatui#13]
Yo all, just wanted to try this out, since it looks like ImGui and I wanted to compare, but I can't even build it:

![flatui](https://cloud.githubusercontent.com/assets/5236548/25101078/26a94de6-23b3-11e7-9c06-906429cf1555.png)

I used CMake GUI to build the project, deselected 2 DirectX checkbuttons, because it didn't find the DX SDK (Windows 10 have it already?). Well, generating .sln worked nicely then, but it missed the generation of these header files it seems.

**Comment by nat42** [comment]
Yeah hit this issue too. These errors came from the gumbo-parser dependency which has it's own include file for strings.h but the FlatUI Cmake gereated project files don't seem to be setup to use it.

After I edited the generated visual studio project to include the gumbo-parser supplied strings.h it compiled.
